### PR TITLE
Display content in packages grid

### PR DESCRIPTION
**Pull request by isaac73**
# Type of change? 

- [ ] Bug fix
- [x] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 
This PR is part of Edge-crawl feature.

This PR has changes related to displaying data in packages grid
...

**Checklist:**

- [x] All tests passed
- [ ] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

---
This change is [Reviewable](https://reviewable.io/reviews/azure/pcs-remote-monitoring-webui/1068)

### CODE REVIEW

File: public/locales/en/translations.json [hunk 1]
```diff
@@ -547,10 +547,16 @@
     "noneFound": "No packages found.",
     "title": "Packages",
     "total": "total packages",
-    "grid": {},
+    "newBtn": "New Package",
```
Comment (mechaffin) [inline public/locales/en/translations.json]: Remove "Btn" from these names.

File: src/components/pages/packages/packages.js [hunk 2]
```diff
@@ -34,26 +34,24 @@ export class Packages extends Component {
...
 
+  onContextMenuChange = contextBtns => this.setState({
+    contextBtns,
+    openFlyoutName: undefined
+  });
+
   openNewPackageFlyout = () => this.setState({
-    openFlyoutName: 'newPackage',
-    selectedPackageId: ''
+    openFlyoutName: 'newPackage'
```
Comment (mechaffin) [inline src/components/pages/packages/packages.js]: Use cabob casing like on other pages. "new-package"

File: src/components/pages/packages/packagesGrid/packagesGrid.js [hunk 3]
```diff
@@ -1,12 +1,92 @@
...
+    if (isFunc(this.props.onGridReady)) {
+      this.props.onGridReady(gridReadyEvent);
+    }
+  };
+
+  /**
+   * Handles context filter changes and calls any hard select props method
+   *
+   * @param {Array} selectedPackages A list of currently selected packages
+   */
```
Comment (mechaffin) [inline src/components/pages/packages/packagesGrid/packagesGrid.js]: check spacing

File: src/components/pages/packages/packagesGrid/packagesGrid.js [hunk 4]
```diff
@@ -1,12 +1,92 @@
...
+      unSortIcon: true,
+      onHardSelectChange: this.onHardSelectChange,
+      context: {
+        t: this.props.t
+      }
+    };
 
-  render (){
-    return [];
+    return ([<PcsGrid {...gridProps} key="rules-grid" />]);
```
Comment (mechaffin) [inline src/components/pages/packages/packagesGrid/packagesGrid.js]: Why are you returning an array?

File: src/services/models/authModels.js [hunk 5]
```diff
@@ -21,7 +21,10 @@ export const permissions = {
 
   createJobs: 'CreateJobs',
 
-  updateSIMManagement: 'UpdateSIMManagement'
+  updateSIMManagement: 'UpdateSIMManagement',
+
+  deletePackages: 'DeletePackages',
+  addPackage: 'AddPackage'
```
Comment (mechaffin) [inline src/services/models/authModels.js]: All of the other permission names are plural. This one should be too.

File: src/components/pages/packages/packagesGrid/packagesGrid.js (hunk 3 above)
Comment (isaac73) [reply src/components/pages/packages/packagesGrid/packagesGrid.js]: It's correct

File: src/components/pages/packages/packagesGrid/packagesGrid.js (hunk 4 above)
Comment (isaac73) [reply src/components/pages/packages/packagesGrid/packagesGrid.js]: fixed

File: src/services/models/authModels.js (hunk 5 above)
Comment (isaac73) [reply src/services/models/authModels.js]: changed

File: src/components/pages/packages/packages.js (hunk 2 above)
Comment (isaac73) [reply src/components/pages/packages/packages.js]: done

File: public/locales/en/translations.json (hunk 1 above)
Comment (isaac73) [reply public/locales/en/translations.json]: done

**Pull request merged**